cmd/containerd-shim-runhcs-v1/task_wcow_podsandbox.go

```diff
@@ -284,6 +284,9 @@ func (wpst *wcowPodSandboxTask) Share(ctx context.Context, req *shimdiag.ShareRe
 
 func (wpst *wcowPodSandboxTask) Stats(ctx context.Context) (*stats.Statistics, error) {
 	stats := &stats.Statistics{}
+	if wpst.host == nil {
+		return stats, nil
+	}
```

Comment:
is this change not part of a different PR?

Reply:
No it was intentional. We don't launch a pod sandbox container for these, but we re-use the WCOW hypervisor logic and the wcow pod sandbox task. So if we don't have a host (e.g. we don't have a UVM), we don't have anything to grab stats for.